schemas/lib.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: lib
+  title: Shared Library
+  group: 2
+  description: These fields contain information about dynamic or static code libraries, including both kernel-mode modules, and process modules.
```

Comment:
Please add a short description here a well. Aim for about 1/2 the size of this description.